.gitignore

```diff
@@ -11,7 +11,6 @@ testbin/*
 bundle/*
 bundle.Dockerfile
 custom-bundle.Dockerfile.pinned
-config/manager/kustomization.yaml
```

Comment:
I thought there is a reason this file is in .gitignore. The image[1] in kustomization.yaml gets updated when you run docker-build/docker-push and push.

[1] https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/manager/kustomization.yaml#L15

Reply:
It does get updated when you do that.  My understanding was that we have a base `config/manager/kustomization.yaml` and that already exists in the repo.  So the `.gitignore` for it was okay because we already have what we need in the repo.  But I suppose the format might change and at some point we might need a new base?

Reply:
Ah, ok. I overlooked the fact that the file is modified by `make bundle`. Thanks for pointing that out.